index.html

```diff
@@ -38,5 +38,85 @@ <h2 class="ocupacion">I'am a Software Developer</h2>
 <section class="ornament">
 </section>
 
+<section class="middle-section">
+
+<div class="my-recent">
+  <p class="recent-title">My Recent Works</p>
+<img class="recent-img" src="pictures/Img Placeholder.svg" alt="">
+<p class="recent-title2 color-dark">Multi-Post Stories</p>
+<p class=" contend color-dark">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+<ul class="color-dark">
+  <li>CSS</li>
+  <li>html</li>
+  <li>boodstrap</li>
+  <li>Ruby</li>
+</ul>
+<a class="proyect-recent see-style" href="#">See Project</a>
+</div>
+
+<div class="works works1 bak-image work-container">
+  <p class="works-title">Profesional Art Printing Data</p>
+<p class="works-contend">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+<ul>
+  <li>html</li>
+  <li>boodstrap</li>
+  <li>Ruby</li>
+</ul>
+<a class="proyect-button" href="#">See Project</a>
+</div>
+
+<div class="works works1 bak-image work-container">
+  <p class="works-title">Profesional Art Printing Data</p>
+<p class="works-contend">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+<ul>
+  <li>html</li>
+  <li>boodstrap</li>
+  <li>Ruby</li>
+</ul>
+<a class="proyect-button" href="#">See Project</a>
+</div>
+
+<div class="works works1 bak-image work-container">
+  <p class="works-title">Profesional Art Printing Data</p>
+<p class="works-contend">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+<ul>
+  <li>html</li>
+  <li>boodstrap</li>
+  <li>Ruby</li>
+</ul>
+<a class="proyect-button" href="#">See Project</a>
```

Comment:
- [ ]  You should style your link according to provided design. You should make sure that you follow the styling: Hove AND Pressed based on design. This will increase the accessibility of your website.
Check from the template :
![image](https://user-images.githubusercontent.com/43877689/200125751-702dcb93-c46a-40af-848b-86af51a824da.png)